docs/document/content/user-manual/shardingsphere-jdbc/usage/governance/yaml.cn.md

```diff
@@ -32,28 +32,30 @@ weight = 2
 以下示例将 ZooKeeper 作为配置中心和注册中心。
 
 ```yaml
-governance:
-  registryCenter:
-      type: Zookeeper
-      namespace: governance_ds
-      serverLists: localhost:2181
+mode:
+  type: Cluster
+  repository:
+    type: ZooKeeper
+    props:
+      namespace: demo_yaml_ds_sharding
+      server-lists: localhost:2181
   overwrite: true
 ```
 
 ```java
-// 创建 GovernanceShardingSphereDataSource
-DataSource dataSource = YamlGovernanceShardingSphereDataSourceFactory.createDataSource(yamlFile);
+// 创建 ShardingSphereDataSource
+DataSource dataSource = YamlShardingSphereDataSourceFactory.createDataSource(yamlFile);
 ```
 
-## 使用 GovernanceShardingSphereDataSource
+## 使用 ShardingSphereDataSource
 
-通过 YamlGovernanceShardingSphereDataSourceFactory 工厂创建的 GovernanceShardingSphereDataSource 实现自 JDBC 的标准接口 DataSource。
+通过 YamlShardingSphereDataSourceFactory 工厂创建的 ShardingSphereDataSource 实现自 JDBC 的标准接口 DataSource。
 可通过 DataSource 选择使用原生 JDBC，或JPA， MyBatis 等 ORM 框架。
```

Comment:
it's not necessary add a whitespace between English Charactor and a `，`

Reply:
yes, i will modify it.